CHANGELOG.md

```diff
@@ -3,6 +3,50 @@ All notable changes to the .NET Upgrade Assistant will be documented in this fil
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+### Added
+- Include try-convert version in upgrade-assistant output [(#358)](https://github.com/dotnet/upgrade-assistant/pull/358)
+- Add check so ensure file can be loaded. [(#346)](https://github.com/dotnet/upgrade-assistant/pull/346)
+- Add a command to select a different current project [(#343)](https://github.com/dotnet/upgrade-assistant/pull/343)
+- Add a step to update package dependecies before TFM change [(#342)](https://github.com/dotnet/upgrade-assistant/pull/342)
+- Add IComponentIdentifier and ProjectComponents.AspNetCore [(#340)](https://github.com/dotnet/upgrade-assistant/pull/340)
```

Comment:
This line can probably be removed because I don't think there's much customer-visible change.

Reply:
Removed this line, good call.